### ISSUE TITLE
Keyboard Backlight Stop Working

### ISSUE THREAD

**Issue by iwindhu** [dortania/OpenCore-Legacy-Patcher#585]
### Machine Model

MacBook8,1

### Application Version

Latest Release

### Application Variant

TUI (Text User Interface)

### What versions of macOS are you seeing the problem on?

macOS 11, Big Sur

### Where does this issue happen?

Other/Non-Applicable

### What is the Isssue?

hi there..i just install macos bigsur 11.6 on macbook pro 13" early 2011 using OLP 0.3.0 but unfortunately after patching, the keyboard backlight stop working and the control setting gone. What should i do to fixed it ? Thanks
![Screen Shot 2021-10-18 at 9 50 37 AM](https://user-images.githubusercontent.com/92450476/137657084-69fc9348-1054-4936-9097-d886ad2ec755.png)
![Screen Shot 2021-10-18 at 9 47 59 AM](https://user-images.githubusercontent.com/92450476/137657099-8cad7823-af63-4fa6-b245-5f288301bd6b.png)
![Screen Shot 2021-10-18 at 9 43 50 AM](https://user-images.githubusercontent.com/92450476/137657107-0e6a2d03-25b3-425a-8f3e-a6d8bd3138b1.png)

.

**Comment by NightRai** [comment]
Any other solution than downloading another app. is there a patch in place to use current short keys

**Comment by emitt234** [comment]
Same here

**Comment by NightRai** [comment]
I found a solution. Download Touché and set it to start at logon. Once computer boots, you can hide or quit the app. It will make the keyboard work properly as normal. Even the functions keys F5 and F6 work. You can also control from drop down on top left and set backlight to turn off after inactivity in system preferences. Enjoy

**Comment by HackintoshHD** [comment]
@NightRai Thank you very much for the info on Touché - that's also reviving the keyboard backlight OSD control on my MacBookPro Mid 2009 (MacBookPro5,2).

**Comment by NightRai** [comment]
It worked on 5,4 but didn't work on my 8,1

**Comment by Ausdauersportler** [comment]
@NightRai 
You may add this finding to the [docs](https://dortania.github.io/OpenCore-Legacy-Patcher/ACCEL.html#keyboard-backlight-broken) - having a GitHub account you can make and propose changes to the docs by using the link on bottom of each page named **Help us improve this page**.

**Comment by NightRai** [comment]
I tried to comment to the page but not sure it didn't accept change.

**Comment by HackintoshHD** [comment]
@NightRai Your [pull request](https://github.com/dortania/OpenCore-Legacy-Patcher/pull/732/commits) does exist, it's just not merged (yet).

Maybe Touché should be even featured before LabTick in the corresponding paragraph considering it's really _restoring_ the macOS keyboard backlight OSD functionality, even after having quit, and not just offering an alternative GUI element for manually setting the brightness of the keyboard backlight?

**Comment by NightRai** [comment]
Yes I think it should be up before. But it doesn't work well with MacBook Pro 8,1

**Comment by hvdschmitt** [comment]
Thank you NightRai for pointing us to Touché. I'm using it on an MBP5,2 (like Hackintosh HD) with BS and Monterey.
